app/src/org/commcare/views/media/MediaLayout.java

```diff
@@ -103,9 +105,14 @@ public static MediaLayout buildComprehensiveLayout(Context context,
                                                        TextView text, String audioURI, String imageURI,
                                                        final String videoURI, final String bigImageURI,
                                                        final String qrCodeContent, String inlineVideoURI,
+                                                       final String ttsText,
                                                        int questionIndex) {
         MediaLayout mediaLayout = new MediaLayout(context);
         mediaLayout.setAVT(text, audioURI, imageURI, videoURI, bigImageURI, qrCodeContent, inlineVideoURI, false, questionIndex);
+        // Show TTS view only when audioURI is not present
+        if (ttsText != null && audioURI == null) {
```

Comment:
How does the layout look when you have TTS plus a videoURI ?

Reply:
![Screenshot_2020-12-15-10-12-39-794_org commcare dalvik debug](https://user-images.githubusercontent.com/29516995/102172170-42c75480-3ebe-11eb-8b27-0d596bb9bb5f.jpg)

Reply:
thanks, this will work.